lib/node_modules/@stdlib/array/base/cusome/docs/types/test.ts

```diff
@@ -0,0 +1,114 @@
+/*
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+
+import cusome = require( './index' );
+
+
+// TESTS //
+
+// The function returns an array...
+{
+	cusome( [ false, false, true, false, false ] ); // $ExpectType boolean[]
```

Comment:
These tests should be failing atm. The invocation signatures are missing the `n` argument. That applies here and below.